lib/rex/post/meterpreter/extensions/stdapi/tlv.rb

```diff
@@ -47,6 +47,10 @@ module Stdapi
 TLV_TYPE_SEARCH_GLOB        = TLV_META_TYPE_STRING  | 1231
 TLV_TYPE_SEARCH_ROOT        = TLV_META_TYPE_STRING  | 1232
 TLV_TYPE_SEARCH_RESULTS     = TLV_META_TYPE_GROUP   | 1233
+TLV_TYPE_SEARCH_MTIME       = TLV_META_TYPE_UINT    | 1235
+TLV_TYPE_SEARCH_FROM_DATE   = TLV_META_TYPE_UINT    | 1236
+TLV_TYPE_SEARCH_TO_DATE     = TLV_META_TYPE_UINT  | 1237
```

Comment:
Since these are applied to the modified timestamp, they should be prefixed as such so that we could hypothetically add a similar search for created timestamps in the future without a bunch of refactoring. Everywhere that we're referring to these values, it should be clear that they're applied to the modified field.

Reply:
I updated to use TLV_TYPE_SEARCH_M_START_DATE and TLV_TYPE_SEARCH_M_END_DATE.  I switched to start/end instead of using from/to in order to be consistent with some other places, like below.